src/controllers/itemDetails/index.js

```diff
@@ -1002,6 +1003,93 @@ function renderStudio(page, item, context) {
     studiosGroup.classList.toggle('hide', !studios.length);
 }
 
+function renderAuthor(page, item, context) {
+    const authors = (item.People || []).filter(function (person) {
+        return person.Type === 'Author';
+    });
+
+    const html = authors.map(function (person) {
+        return '<a style="color:inherit;" class="button-link" is="emby-linkbutton" href="' + appRouter.getRouteUrl({
+            Name: person.Name,
+            Type: 'Person',
+            ServerId: item.ServerId,
+            Id: person.Id
+        }, {
+            context: context
+        }) + '">' + escapeHtml(person.Name) + '</a>';
+    }).join(', ');
+
+    const authorsLabel = page.querySelector('.authorsLabel');
+    authorsLabel.innerHTML = globalize.translate(authors.length > 1 ? 'Authors' : 'Author');
+    const authorsValue = page.querySelector('.authors');
+    authorsValue.innerHTML = html;
+
+    const authorsGroup = page.querySelector('.authorsGroup');
+    if (authors.length) {
+        authorsGroup.classList.remove('hide');
+    } else {
+        authorsGroup.classList.add('hide');
+    }
```

Comment:
You can use `toggle`.
```suggestion
    authorsGroup.classList.toggle('hide', !authors.length);
```